API.md

```diff
@@ -3,17 +3,21 @@
 ### Table of Contents
```

Comment:
Don't mind the markdown changes, I used a new version of documentation.js, which led to the diff.